Project/src/main/java/com/github/lgooddatepicker/components/CalendarPanel.java

```diff
@@ -300,6 +300,15 @@ public void integerTextFieldNumberChanged(JIntegerTextField source, int newValue
                 drawCalendar(newYearMonth);
             }
         };
+        // Code passes Maven's tests and compiles successfully.
+        // Issue #113: Pressing enter does the same as pressing the doneEditingYearButton
+        yearTextField.addActionListener(new ActionListener() {
+                @Override
+                public void actionPerformed(ActionEvent e) {
+                    // This should behave the same as the doneEditingYearButton
```

Comment:
This comment is essentially a repetition of the one in line 304, please remove it.